src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tabId: string) => void
+}
+
+export const Tabs = (props: Props) => {
+  const { tabs, selectedTabId, onTabSelected } = props;
+  const content = tabs.find(tab => tab.id === selectedTabId)?.content;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <li
```

Comment:
```suggestion
          {tabs.map(tab => (
            <li
```
you can do it in this way